src/main/resources/assets/css/marathon.css

```diff
@@ -161,6 +161,10 @@ a,
   margin-right: 10px;
 }
 
+.list-inline > li:last-child {
+  margin-right: 0px;
```

Comment:
Loose the px and add remove the extra padding as well, and let's keep it with tables only (so maybe also move it up a nudge to the other table specific styles):
```
.table > thead > tr > td > .list-inline > li:last-child,
.table > thead > tr > th > .list-inline > li:last-child {
  margin-right: 0;
  padding-right: 0;
}
```